srtp/srtp.c

```diff
@@ -1482,6 +1786,16 @@ srtp_unprotect_aead (srtp_ctx_t *ctx, srtp_stream_ctx_t *stream, int delta,
     }
   }
 
+  if (xtn_hdr && stream->rtp_xtn_hdr_cipher) {
+    /*
+     * extensions header encryption RFC 6904
+     */
+    status = srtp_process_header_encryption(stream, xtn_hdr);
```

Comment:
This looks good.  You may need to re-submit the pull request, this commit isn't showing in the pull request.